ivi-layermanagement-examples/LayerManagerControl/src/print.cpp

```diff
@@ -29,7 +29,7 @@ using std::endl;
 using std::dec;
 using std::hex;
 
-
+#include <stdlib.h>
```

Comment:
Thanks for this. I believe I've run into compilation failure at some point when free() was seen as an undeclared function.